### DISCUSSION TITLE
Article link replacement - possible?

### DISCUSSION THREAD

**Discussion by pniewiadowski** [FreshRSS/FreshRSS#7136] [Q&A]
I use FreshRSS and I pull RSS from wykop.pl, for example: https://wykop.pl/api/v3/dmz/rss/tags/druk3d
Sample entry:
```
<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0">
<channel>
<atom:link href="https://wykop.pl/rss/tags/druk3d" rel="self" type="application/rss+xml"/>
<link>http://wykop.pl</link>
<pubDate>Tue, 10 Dec 2024 13:57:02 +0100</pubDate>
<title>#druk3d :: Wykop.pl</title>
<description>Wykop - service created by users</description>
<item>
<title>Anycubic Kobra 3 Combo for 1650PLN Worth it or not</title>
<link>https://wykop.pl/entry/79323161/anycubic-kobra-3-combo-za-1650zl-warto-czy-nie-war</link>
<guid>https://wykop.pl/entry/79323161/anycubic-kobra-3-combo-za-1650zl-warto-czy-nie-war</guid>
<pubDate>Tue, 10 Dec 2024 13:57:02 +0100</pubDate>
<description>
<![CDATA[ Anycubic Kobra 3 Combo for 1650PLN Worth it or not? #3dprint #3dprinters #asktheexpert ]]>
</description>
</item>
```
Unfortunately, the <link> has "entry" in it but it should have "wpis".

Are there any option to replace the "entry" with wpis on article level so when I click on article it redirect me to the right page?
`https://wykop.pl/entry/(...)` -> `https://wykop.pl/wpis/(...)`

PS. I know it should be fixed on wykop.pl site, but It is like this for almost 2 years and nobody cares about it.

**Comment by math-GH** [comment]
> PS. I know it should be fixed on wykop.pl site, but It is like this for almost 2 years and nobody cares about it.

Have you contacted them?

What you can do with FreshRSS:
I guess it would possible with an extension.

Another solution could be to scrap your own feed with the build-in web scrapper.
